src/App.tsx

```diff
@@ -1,18 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
+export const options = ['3', '5', '10', '20'];
+
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<string>(options[1]);
+  const [currentPage, setCurrentPage] = useState(0);
+  const countPage = Math.ceil(items.length / Number(perPage));
+
+  const handleChangCountItem = (
```

Comment:
```suggestion
  const handleChangeCountItem = (
```

Reply:
Not resolved. Fix spell:

Chang -> Change